incubator/extpipes_cli/__init__.py

```diff
@@ -1,9 +1 @@
-# version
```

Comment:
How to keep this `__version__` in sync with the semantic-versioning?
Let's check this next round